modules/ROOT/pages/7.0.1-release-notes.adoc

```diff
@@ -78,13 +78,44 @@ The {productname} {release-version} release includes an accompanying release of
 
 This **Table of Contents** release includes the following fixes.
 
-==== Encode header title for literal display in table of contents.
+==== Encoding header of Table of Contents causing HTML source to be revealed on refresh.
+// #TINY-10682
+
+Previously in {productname}, the **Table of Contents** (ToC) header was encoded on refresh before being displayed in the editor. This resulted in the header being incorrectly shown as the underlying code literally, instead of being rendered as HTML.
+
+To address this issue, the HTML encoding has been removed. As a result, the ToC header now renders correctly as HTML even after refreshing, displaying as intended.
+
+For example, if the **Table of Contents** (ToC) header contained the following content with "_Contents_" emphasized as follows:
+
+*`Table of _Contents_`*
+
+Upon refreshing the ToC, the header would change to display the HTML tags literally, as follows:
+
+*`Table of <em>Contents</em>`*
+
+Now, the header maintains the intended HTML rendering even after refreshing:
+
+*`Table of _Contents_`*
+
+==== Encode header title for literal display in Table of Contents.
 // #TINY-10789
 
 Previously in {productname}, during Table of Contents (ToC) generation, heading elements containing HTML tags such as `<i>` for italics were incorrectly rendered within the ToC itself.
 
 {productname} {release-version} addresses this issue by encoding the heading element during ToC generation. This ensures that any HTML tags present within the heading element are treated as plain text and displayed literally.
 
+For example, if a heading element contained the following content:
+
+*`My <em>Heading</em>`*
+
+Upon inserting or refreshing the ToC, the heading element would be rendered as HTML, resulting in the following entry:
+
+*`Table of _Contents_`*
```

Comment:
Should we include the ToC header and the content or the heading element inside the ToC? The change affects the heading within the ToC, not sure how do we put it here though.